src/domTree.js

```diff
@@ -125,7 +125,31 @@ const toMarkup = function(tagName: string): string {
     return markup;
 };
 
-export type CssStyle = {[name: string]: string};
+// Making the type below exact with all optional fields doesn't work due to
+// - https://github.com/facebook/flow/issues/4582
+// - https://github.com/facebook/flow/issues/5688
+// However, since *all* fields are optional, $Shape<> works as suggested in 5688
+// above.
```

Comment:
Could you add a comment this is not comprehensive?

Reply:
I amended the commit to include `// Optional settings for styling.` on the line above this snippet (included in the current submission, though you'll have to view the rest of the file to confirm)---does that suffice?

Reply:
OK, I've added your recommendation to the line just below this snippet.